### PR TITLE
layout_catalog

### PR DESCRIPTION
**Pull request by Tarasov123555**
- [DEMO LINK](https://Tarasov123555.github.io/layout_catalog/)
- [TEST REPORT LINK](https://Tarasov123555.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,8 +5,365 @@
...
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo"><img src="images/logo.png" alt="/"></a>
```
Comment (vlad-demchuk) [inline src/index.html:14]: set descriptive alt attribute

File: src/index.html [hunk 2]
```diff
@@ -5,8 +5,365 @@
...
+      </nav>
+    </header>
+
+    <main class="content">
+      <ul class="content__list">
+        <div class="cards">
+          <container class="product" data-qa="card">
+            <img class="product__image"
+         src="images/imac.jpeg"
+         alt="APPLE A1419 iMac 27">
```
Comment (vlad-demchuk) [inline src/index.html:75]:
```suggestion
            <img 
              class="product__image"
              src="images/imac.jpeg"
              alt="APPLE A1419 iMac 27"
            >
```

File: src/index.html [hunk 3]
```diff
@@ -5,8 +5,365 @@
...
+
+            <h2 class="product__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
```
Comment (vlad-demchuk) [inline src/index.html:79]:
```suggestion
            <h2 class="product__name">
              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
            </h2>
```
check your code style
fix all cases

File: src/index.html [hunk 4]
```diff
@@ -5,8 +5,365 @@
...
+            </p>
+
+            <a href="#"
+        class="product__buy-btn"
+        data-qa="card-hover">
+        Buy
+            </a>
+          </container>
+
+          <container class="product">
```
Comment (vlad-demchuk) [inline src/index.html:109]: have I missed something? is `container` a new tag?

File: src/styles/blocks/header.scss [hunk 5]
```diff
@@ -0,0 +1,12 @@
+.header {
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+
+  &logo {
```
Comment (vlad-demchuk) [inline src/styles/blocks/header.scss:8]:
```suggestion
  &__logo {
```

if it is inside bem block, it should be a bem element

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 6]
```diff
@@ -5,8 +5,378 @@
...
+        Buy
+            </a>
+          </div>
+
+          <div class="product">
+            <img
+         class="product__image"
+         src="images/imac.jpeg"
+         alt="APPLE A1419 iMac 27"
+         >
```
Comment (VitaliyBondarenko1982) [inline src/index.html:117]:
```suggestion
            <img
              class="product__image"
              src="images/imac.jpeg"
              alt="APPLE A1419 iMac 27"
            >
```

File: src/index.html [hunk 7]
```diff
@@ -5,8 +5,378 @@
...
+
+            <h2 class="product__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+
+            <p class="product__code">
+           Product code: <span>195434</span>
+            </p>
```
Comment (VitaliyBondarenko1982) [inline src/index.html:125]:
```suggestion
            <p class="product__code">
              Product code: <span>195434</span>
            </p>
```

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Good job.
Just pay attention on code style in next tasks

**Review by VitaliyBondarenko1982 — APPROVED** [review]
Good job.
Just pay attention on code style in next tasks

**Pull request closed without merging**